libs/platform/languages.hpp

```diff
@@ -39,12 +39,16 @@ std::array<std::pair<std::string_view, std::string_view>, 41> constexpr kLanguag
 #else
     {"es", "Español"},       {"es-MX", "Español (México)"},
 #endif
+    {"et", "Estonian"},
     {"eu", "Euskara"},
     {"fr", "Français"},
+    {"gl", "Galician"},
     {"hr", "Hrvatski"},
     {"it", "Italiano"},
     {"sw", "Kiswahili"},
+    {"he", "Hebrew"},
```

Comment:
Should be after uk and before ar
```suggestion
    {"he", "עברית"},
```